arch/esp32/esp32c6.ini

```
@@ -32,6 +30,7 @@ lib_deps =
   https://github.com/meshtastic/ESP32_Codec2/archive/633326c78ac251c059ab3a8c430fcdf25b41672f.zip
   # renovate: datasource=custom.pio depName=rweather/Crypto packageName=rweather/library/Crypto
   rweather/Crypto@^0.4.0
+  https://github.com/h2zero/esp-nimble-cpp.git#2.2.1
```

Hate to be a party pooper, but these GIT pulls are very expensive during build time so we replaced them with on-the-fly ZIP files. Please consider using https://github.com/h2zero/esp-nimble-cpp/archive/2.2.1.zip instead

---

```suggestion
  # renovate: datasource=github-tags depName=esp-nimble-cpp packageName=h2zero/esp-nimble-cpp
  https://github.com/h2zero/esp-nimble-cpp/archive/2.2.1.zip
```
Here's a complete version of what @caveman99 described :heart: with renovate

---

Yep, improved that part. Thank you!